src/ImageSharp/Formats/Gif/GifConstants.cs

```diff
@@ -90,6 +91,16 @@ internal static class GifConstants
         /// <summary>
         /// Gets the default encoding to use when reading comments.
         /// </summary>
-        public static Encoding DefaultEncoding { get; } = Encoding.GetEncoding("ASCII");
+        public static readonly Encoding DefaultEncoding = Encoding.GetEncoding("ASCII");
```

Comment:
Left it separate for now, its safer in case we want to switch out the encoder to be different based on format. I favour having the formats as isolated from each other as we can. Less chance on causing cross encoder bugs during fixes then.